.vscode/launch.json

```diff
@@ -0,0 +1,26 @@
+{
+    // Use IntelliSense to learn about possible attributes.
+    // Hover to view descriptions of existing attributes.
+    // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
+    "version": "0.2.0",
+    "configurations": [
+        {
+            "type": "node",
+            "request": "launch",
+            "name": "Launch P0 CLI",
```

Comment:
This name is misleading, as this actually uses the P0 CLI to SSH to a specific VM

Reply:
I removed that.